notes_app/client/app/users/users_list.component.js

```diff
@@ -4,15 +4,20 @@
   UsersCtrl.$inject = ['userService', '$scope'];
   function UsersCtrl(userService, $scope) {
     var ctrl = this;
+
     ctrl.users = [];
+    ctrl.currentPage = 1;
+
+
     ctrl.gridOptions = {
       onRegisterApi: function( gridApi ) {
         ctrl.gridApi = gridApi;
         gridApi.rowEdit.on.saveRow($scope, ctrl.saveRow);
       },
       data: [],
-      paginationPageSize: 5,
-      paginationPageSizes: [5,10,25],
+      enablePaginationControls: false,
+      paginationPageSize: 25,
+//      paginationPageSizes: [5,10,25],
```

Comment:
please don't leave commented out code. Usually it's not needed, but if you need this - leave an explanation around